mne/utils.py

```diff
@@ -1205,6 +1205,7 @@ def set_memmap_min_size(memmap_min_size):
     'MNE_SKIP_TESTING_DATASET_TESTS',
     'MNE_STIM_CHANNEL',
     'MNE_USE_CUDA',
+    'MNE_TESTING',
```

Comment:
We have others of the form `MNE_SKIP_*`, so maybe `MNE_SKIP_FLASH_CALL` or something would be better.

Although in this case, we want the default behavior when running `nosetests mne/tests/test_bem.py` to skip the FreeSurfer call, since it takes forever. Your current code will still make the call to FreeSurfer by default (although you have made AppVeyor and Travis skip it, *developers* will still be haunted by the slowness).